pandas/tests/indexes/test_multi.py

```diff
@@ -3291,3 +3291,29 @@ def test_duplicate_multiindex_labels(self):
         with pytest.raises(ValueError):
             ind.set_levels([['A', 'B', 'A', 'A', 'B'], [2, 1, 3, -2, 5]],
                            inplace=True)
+
+    @pytest.mark.parametrize("midx,idx,count", [
+        (pd.MultiIndex.from_product([[0, 1], [1, 0]]), pd.Series(range(4)), 4),
+        (pd.MultiIndex.from_product([[0, 1]]), pd.Series(range(2)), 2)])
+    def test_multiindex_compare(self, midx, idx, count):
+        # GH 21149
+        '''Ensure comparison operations for MultiIndex with nlevels == 1
+            behave consistently with those for MultiIndex with nlevels > 1
+        '''
+        expected = pd.Series([True]).repeat(count)
+        expected.reset_index(drop=True, inplace=True)
```

Comment:
don't use inplace in tests

Reply:
@jreback Thanks - here `inplace` is being used to only create the expected outcome for testing. `inplace` is not being used in the test `assert_series_equal`
OK to keep then?